include/myst/thread.h

```diff
@@ -208,6 +209,7 @@ struct myst_thread
 
     /* used by myst_thread_queue_t (condition variables and mutexes) */
     struct myst_thread* qnext;
+    uint64_t bitset;
```

Comment:
Should be `uint32_t since `bitset` is `uint32_t` elsewhere and the mask itself is 32-bits.